CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### App Center
 
 * **[Fix]** Handle incorrect usage of `AppCenter.setLogUrl` API to provide readable error message.
+* **[Fix]** Check for running in App Center Test will now work on newer Android SDKs.
```

Comment:
```suggestion
* **[Fix]** Check for running in App Center Test will now work when using AndroidX instead of the support library.
```
This is not directly related to Android version but rather which test framework being used.